installer/common/detect.sh

```diff
@@ -53,30 +53,30 @@ detect_kafkalagexporter() {
 
 # Returns a zero if kafka has not been deployed into this namespace
 detect_kafka_cluster() {
-	kubectl get deployment -l strimzi.io/kind=Kafka --ignore-not-found=true -n "$1" --no-headers | wc -l
+    kubectl get deployment -l strimzi.io/kind=Kafka --ignore-not-found=true -n "$1" --no-headers | wc -l
```

Comment:
Since I am not sure about the space consistency for the rest of the files of the code base, could you please revert changes that modify indentation. I will create an issue for this to reduce our technical debt.

Reply:
@skonto Usually we use a 2 space indentation though it's not enforced anywhere through a tool ..